pkg/domain/attributor.go

```diff
@@ -141,6 +141,20 @@ func (err AssetInventoryMultipleAssetsFoundError) Error() string {
 		err.AssetInventory, err.AssetID, err.ScanTimestamp, err.Inner)
 }
 
+// AssetInventoryMultipleAttributionErrors occurs when multiple attribution errors
+// occur on multiple attribution sources
+type AssetInventoryMultipleAttributionErrors struct {
+	Inner         error
+	AssetID       string
+	ScanTimestamp string
+}
+
+func (err AssetInventoryMultipleAttributionErrors) Error() string {
```

Comment:
Suggestion: Similar to the change for PSD-191, I'd like for us to move toward a static `Reason` field, and we can log the other pieces, e.g., `AssetID`, `ScanTimestamp`, and even `Inner` as additional fields in the structured error log line emitted from the handler code.

Reply:
done! err... resolved